macros/materializations/dummy.sql

```diff
@@ -0,0 +1,10 @@
+{# Materialization that does not create any table at the end of its run #}
+{# An example for a usage case is when we want a module to appear on the Elementary lineage graph, but the table is created outside of dbt #}
+{% materialization dummy, default -%}
+  {# The main statement executes the model, but does not create any table / view on the DWH #}
```

Comment:
Why do we execute anything at all in the dummy materialization?

Reply:
@elongl  this part executes the sql from the model definition, which is supposed to be empty except for the config and the dependencies, but we do need those

Reply:
@haritamar I copied this from the CLI repo and didn't change anything, maybe for other use cases we do need it? let's discuss tomorrow :)